src/index.html

```diff
@@ -20,8 +21,424 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#MoyoLogo"
+        class="logo"
```

Comment:
logo is an element of header BEM block, pls use proper classname according to the BEM naming convention